SYS-1986: Add `media_type` field to `SheetImport` objects

henry-r18 opened the pull request:
Implements [SYS-1986](https://uclalibrary.atlassian.net/browse/SYS-1986)

### Acceptance criteria
- [x] Field `media_type` is added with `max_length` set to 10 and not required
  - [x] Values in this order, no default: video, audio, image 
- [x] Field appears below `file_type` in Basic Fields, for both view and edit

### Description
This PR adds a `media_type` field to the `SheetImport` objects. Necessary changes are made to the model, view utilities, and form.

One note about the migration file is that when I generated it via `python manage.py makemigrations` in my local instance, Django included a migration operation to alter the UUID on the `HistoricalSheetImport` model. I removed that operation from the auto-generated migration file, in order to keep things discrete. I figured we can address the `HistoricalSheetImport` alteration in a different PR, if necessary, though I'm not exactly sure what the benefit would be.

### Screenshot
<img width="2538" height="1243" alt="image" src="https://github.com/user-attachments/assets/2c22f723-74d7-4f8d-a300-9013384e7106" />


[SYS-1986]: https://uclalibrary.atlassian.net/browse/SYS-1986?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

akohler reviewed (changes requested):
@henry-r18 This PR as-is is fine: field is added, displays and edits correctly.  Please add release notes and bump the version for deployment.  I'd like it to be in production so that values can be added for the 25-50 record pilot.

Please go ahead and add the "Alter field uuid on historicalsheetimport" migration, as a separate file.  Otherwise, Django's just going to keep generating it and I'd rather have it as a separate migration, but it's not worth a separate branch and PR.

This extra migration is because the changes made in #71 did not include a corresponding update to the `historicalsheetimport` model - which was OK, it was a quick bug fix.  And as you noted on that, UUIDs are intended to be unchanging, so history tracking is not relevant... but it's fine and appropriate to make these consistent again, and that'll keep the underlying history model happy.

Thanks --Andy

henry-r18 commented:
@akohler Makes sense re: the `HistoricalSheetImport` migration. I just pushed a new migration for it, and removed the unused `import uuid` from the `media_field`-related migration.

Bumped chart and added release notes. Thanks!

akohler reviewed (approved):
@henry-r18 Looks great now, merging --Andy
@ztucker4 FYI
